src/Compilers/CSharp/Portable/Compilation/SyntaxTreeSemanticModel.cs

```diff
@@ -91,40 +91,32 @@ public override bool IgnoresAccessibility
             get { return _ignoresAccessibility; }
         }
 
-        private void VerifySpanForGetDiagnostics(TextSpan? span)
```

Comment:
I assume you noticed that this was dead code? (no callers of this private method?)